lib/krikri/engine.rb

```diff
@@ -56,5 +56,32 @@ class Engine < ::Rails::Engine
         .register(:oai, Krikri::Harvesters::OAIHarvester)
     end
 
+    initializer :rdf_repository do
+      Krikri::Repository =
+        RDF::Marmotta.new(Krikri::Settings['marmotta']['base'])
+    end
+
+    initializer :aggregation do
+      DPLA::MAP::Aggregation.class_eval do
+        include Krikri::LDP::RdfSource
+        configure :base_uri => Krikri::Settings['marmotta']['item_container']
```

Comment:
Which part in particular?

It's necessary to reopen the class to `include Krikri::LDP::RdfSource`, which isn't necessarily desirable in the broader `dpla-map` context.  The rest of this could conceivably be moved upstream.